driver-testsuite/tests/Basic/CookieTest.php

```diff
@@ -45,6 +45,10 @@ public function testCookie()
         $this->getSession()->setCookie('srvr_cookie', null);
         $this->getSession()->reload();
         $this->assertContains('Previous cookie: NO', $this->getSession()->getPage()->getText());
+
+        $this->getSession()->setCookie('semicolon', 'foo;bar;baz');
+        $this->getSession()->visit($this->pathTo('/cookie_page4.php'));
```

Comment:
I guess theoretically we could reuse vendor/behat/mink-browserkit-driver/tests/web-fixtures/cookie_page2.php:14 ?